src/electrumx/server/block_processor.py

```
@@ -796,14 +826,15 @@ def advance_txs(self, txs, is_unspendable):
 
         # Spend the inputs
         # A separate for-loop here allows any tx ordering in block.
-        for (tx, tx_hash), hashXs in zip(txs, hashXs_by_tx):
+        for tx, hashXs in zip(txs, hashXs_by_tx):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)